### ISSUE TITLE
Contentpack errors on 0.17.0b2

### ISSUE THREAD

**Issue by radinamatic** [learningequality/ka-lite#5336]
## Summary

Some contentpack errors on 0.17 beta release.

## System information

 - Operating system: Ubuntu 16.10 64bit
 - Version: 0.17.0b2
 - Browser: Firefox

## Screenshots

Alert to update language pack right after installation, so far so good...

![0 17-cps1](https://cloud.githubusercontent.com/assets/1457929/20316157/0b7e0068-ab61-11e6-9c8d-fd21ecb8f088.png)

But no way to update once Admin is on Languages page:

![0 17-cps2](https://cloud.githubusercontent.com/assets/1457929/20316181/2219b538-ab61-11e6-9b5a-178d7783a42c.png)

Also, `Delete` button is a bit misnomer here: why would somebody delete the **only** contentpack/language installed...? 

Installed other languages, they all present the alert to update at login, but result apparently "Up-to-date":

![0 17-cps6](https://cloud.githubusercontent.com/assets/1457929/20316553/a255e5b8-ab62-11e6-82bc-771c75c79208.png)

![0 17-cps5](https://cloud.githubusercontent.com/assets/1457929/20316319/a13719a0-ab61-11e6-9df4-6eb93f083381.png)

No `pt-PT` contentpack that I did see yesterday after installing 0.16.9 on Windows XP:

![0 17-cps3](https://cloud.githubusercontent.com/assets/1457929/20316193/28f21e2c-ab61-11e6-98ff-fadf00102b20.png)

Swahili, Zulu & Xhosa language packs have no available videos for download, and load an empty topic tree in the sidebar:

![0 17-cps4](https://cloud.githubusercontent.com/assets/1457929/20316221/4f8d8062-ab61-11e6-920a-37a79d842f9e.png)

#### No available subtitles anywhere!!!

Not sure if this should go in the separate issue?

**Comment by benjaoming** [comment]
Regarding the delete button, please see #5232  and #4864

**Comment by benjaoming** [comment]
All of the content packs are just copied raw from 0.16, I think (but haven't investigated) that this is somehow related, i.e. the old version tag in the JSON metadata prompts the UI javascript to assume an upgrade is available.

**Comment by benjaoming** [comment]
...but we should start building new content packs for 0.17 testing, this could be done as soon as we ensure the current state of things.. I don't feel too on top of the latest content pack updates for various language bugs and whether they are done. But we could also decide to move all efforts to 0.17 language packs.

**Comment by radinamatic** [comment]
@mrpau-eduard @mrpau-richard What say you?

**Comment by mrpau-richard** [comment]
I think we need to build all the language packs and update its version to 0.17.

**Comment by benjaoming** [comment]
@mrpau-richard build? which languages are you talking about here?

Because of many issues, we've decided to *not* build any new versions of content packs that are already working.. it seems fine to copy them straight from 0.16, but please do say if there are any issues?

**Comment by mrpau-richard** [comment]
@benjaoming Yeah, I agree as we discuss in our syncup.

**Comment by radinamatic** [comment]
macOS Sierra, today's OSX installer by @mrpau-richard.

Ok, so upgrading English language, weather from the Terminal or UI, does not register or reflect correctly in Manage > Languages, meaning the button does not change to `Up to date` once the upgrade is finished, and the prompt/alert is still visible on the homepage.

After upgrading from the Terminal, refreshing, re-logging-in and restarting the server:

![virtualbox_sierra_01_12_2016_14_20_08](https://cloud.githubusercontent.com/assets/1457929/20796283/ce0a44ae-b7d5-11e6-9e8f-4ad226b160f9.png)

After re-trying the upgrade from UI and reloading:

![virtualbox_sierra_01_12_2016_14_47_07](https://cloud.githubusercontent.com/assets/1457929/20796301/e9d63292-b7d5-11e6-9e33-3a0e7d7bc9d9.png)

Admin is still prompted to upgrade:

![virtualbox_sierra_01_12_2016_14_50_10](https://cloud.githubusercontent.com/assets/1457929/20796327/008f0298-b7d6-11e6-9af6-d671ef7c9584.png)

@benjaoming If this is not a contentpack error, I'll file it as a separate issue.

**Comment by radinamatic** [comment]
Confirmed with the Spanish contentpack install, after downloading, upgrading and restarting the server, the button still says `Upgrade`: 

![virtualbox_sierra_01_12_2016_15_10_37](https://cloud.githubusercontent.com/assets/1457929/20796864/a48eb378-b7d8-11e6-87d9-7c13613df758.png)

**Comment by benjaoming** [comment]
Closing this, as stated the "Upgrade" buttons are supposed to be displayed because our testing data for 0.17 content packs have correctly been equipped with a meta data saying they are from 0.16...which they are :)

**Comment by radinamatic** [comment]
Ok, will re-test this with the new contentpacks!
